rails/app/assets/stylesheets/components/card.scss

```diff
@@ -20,8 +20,10 @@ $el-shadow: 0 1px 4px #bbb;
   }
 
   &.offCanvas {
+    max-width: 2rem;
+
     .card {
-      left: -100%;
+      display: none;
```

Comment:
Would you mind updating this to use a wrapper element that has 0 width/height and `overflow:hidden`, then set the card element to `visibility:hidden` instead of `display:none`? This should help us a bit with performance since it won't cause a reflow of the DOM. 

If this method gives you trouble though I'm fine with keeping it as `display:none` since that works too and if we don't notice a performance hit then we can probably get away with it :)

Reply:
Thought in reference to your issue getting around the 2rem of area that's not zoomable. Is this because the HTML element is in the way of mouse clicks? If so, could we leverage `pointer-events:none` to allow the click to fall through to the element below (which should be the map)? https://developer.mozilla.org/en-US/docs/Web/CSS/pointer-events